public/index.html

```diff
@@ -24,7 +24,15 @@
       work correctly both with client-side routing and a non-root public URL.
       Learn how to configure a non-root public URL by running `npm run build`.
     -->
-    <title>React App</title>
+    <link rel="apple-touch-icon" sizes="180x180" href="/apple-touch-icon.png">
```

Comment:
Feel free to add open graph meta tags into your project for social media thumbnails and webcrawlers